### PR TITLE
add announcement block (and announcement for mesh.101)

### PR DESCRIPTION
**Pull request by dcwalk**
@garrying and @benhylau any quick thoughts or fixes?

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -46,3 +46,12 @@ $on-laptop:        800px;
 @import "base";
 @import "layout";
 @import "font-awesome";
+
+// announcement
+
+.announcement {
+  background-color: #fffbcc;
+  color: #777;
```
Comment (garrying) [inline css/main.scss:54]: Finding this could use more contrast. Maybe `#333`?

File: css/main.scss [hunk 2]
```diff
@@ -46,3 +46,12 @@ $on-laptop:        800px;
...
+  font-size: 0.8em;
```
Comment (garrying) [inline css/main.scss:55]: Could be bumped up to 0.8 or 1?

**Comment by garrying** [comment]
This looks good to me. Left 2 minor points.

**Comment by benhylau** [comment]
Merging, since this is happening like tomorrow.

**Comment by dcwalk** [comment]
@benhylau thanks, sorry, got caught up with other obligations!

**Comment by benhylau** [comment]
@dcwalk no worries thanks for putting this up!

**Pull request merged**